scripts/kconfig/kconfigfunctions.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2018 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import os
+import re
+
+# Types we support
+# 'string', 'int', 'hex', 'bool'
+conf_file = os.path.join(os.environ['PROJECT_BINARY_DIR'], "include/generated/generated_dts_board.conf")
```

Comment:
assumption is that this plugin will be used for other things, not only DTS, so your globals here need to be namespaced and be specific to DTS and not as generic as they are now.

Reply:
Should be fixed.